src/org/pushingpixels/substance/internal/ui/SubstanceTableUI.java

```diff
@@ -993,8 +994,10 @@ protected void paintCell(Graphics g, Rectangle cellRect, Rectangle highlightCell
             rendererComponent = this.table.prepareRenderer(renderer, row, column);
             boolean isSubstanceRendererComponent = isSubstanceDefaultRenderer(rendererComponent);
             if (isSubstanceRenderer && !isSubstanceRendererComponent) {
-                throw new IllegalArgumentException(
+            	if (!Boolean.getBoolean("substance.looseTableCellRenderers")) {
```

Comment:
This is not going to make it in.

Reply:
The following issue let me create this pull request
https://github.com/Insubstantial/insubstantial/issues/63

Reply:
I'm not going to comment on decisions made in the project fork.

If this is an issue in your app, it needs to be addressed properly - in the context of the planned enhancement for https://github.com/kirill-grouchnikov/substance/issues/66

There will be no one-off properties like this in the codebase.